kalite/contentload/management/commands/channels/khan.py

```diff
@@ -139,6 +139,12 @@ def build_full_cache(items, id_key="id"):
         except AttributeError:
             logging.error("Unable to serialize %r" % item)
 
+        item = whitewash_node_data(item)
+
+        if ids:
+            if item["id"] not in ids:
```

Comment:
I think id_key is actually redundant at this point, it was being used previously to double check things prior to whitewash_node_data being applied, but after whitewash_node_data, every node has 'id' as its id_key.